capsules/extra/src/test/hmac_sha256.rs

```diff
@@ -81,3 +82,8 @@ impl digest::ClientHash<32> for TestHmacSha256 {
         kernel::debug!("HMAC-SHA256 matches!");
     }
 }
+
+impl digest::ClientVerify<32> for TestHmacSha256 {
+    fn verification_done(&self, _result: Result<bool, ErrorCode>, _compare: &'static mut [u8; 32]) {
+    }
+}
```

Comment:
Why did this empty `verification_done()` implementation get added? Is this related?

Reply:
Yes. We need this so that the `Digest` trait becomes implemented for `T` (`TestHmacSha256`), and therefore we can call the correct `set_client()` function.

In case you are wondering what happens to tests when they are not compiled & executed on any board...

Reply:
Oh, I see. It was calling the incorrect `set_client` before.